packages/vite-plugin-cloudflare/playground/spa-with-api/__tests__/spa-with-api.spec.ts

```diff
@@ -18,3 +19,27 @@ test("returns the response from the API", async () => {
 	const contentAfter = await button.innerText();
 	expect(contentAfter).toBe("Name from API is: Cloudflare");
 });
+
+test("returns the home page even for 404-y pages", async () => {
+	page.goto("/foo");
```

Comment:
These tests are failing at the moment because the navigations need full URLs and to be awaited:

```suggestion
	await page.goto(`${viteTestUrl}/foo`);
```